packages/extra/ingress/Makefile

```diff
@@ -1,2 +1,12 @@
+get-cloudflare-ips:
+	curl -s https://www.cloudflare.com/ips-v4/# | grep -oE '([0-9]{1,3}\.){3}[0-9]{1,3}/[0-9]{1,2}' > ips-v4.tmp
```

Comment:
I would recommend adding a timeout, for example 2-4 seconds, so as not to wait 30 or more seconds if this resource is unavailable for one reason or another.